content/en/docs/administration-guide/managing-network-configuration/configuring-a-secondary-network-interface-using-cloud-network.md

```diff
@@ -0,0 +1,148 @@
+---
+title:  Configuring a Secondary Network Interface using Cloud-Network 
+weight: 17
+---
+
+When you add a secondary network interface to a linux instance in the cloud environment, you need to configure the network parameters for the secondary interface in the linux instance. The configuration ensures that you do not face any routing issues while using the secondary network interface. Configuring the secondary network interface involves several manual processes that include configuring a new routing table, setting up rules in the routing table and so on.
+
+`cloud-network` automates the whole manual process of configuring the secondary network interface. It configures the network parameters required for any network interfaces that you create or add to the linux instance. In a cloud environment, instances are set to public IPs and private IPs. If you add more than one private IP for the secondary network interface, the IP other than the one provided by DHCP cannot be fetched and configured for your virtual machine. The `cloud-network` project is designed to adapt the cloud-network environments such as Azure, GCP, and Amazon EC2. `cloud-network` fetches the metadata from the metadata server endpoint, parses the metadata, and then assigns IPs and routes. When `cloud-network` is installed, it automatically configures network interfaces in the cloud frameworks. It detects the available instances using netlink. Additionally, for all the interfaces, including the primary one, it looks for any secondary IP addresses from the metadata server endpoint and configures them on the interface, if any. 
+
```

Comment:
It detects the available instances using netlink. -> It detects the available interfaces using netlink.